File: timus/java/p1636.java
import java.util.Scanner;

/**
 * Penalty Time
 * @author Milena Araujo
 *
 */

public class p1636 {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int t1 = input.nextInt();
		int t2 = input.nextInt();
		
		for (int i = 0; i < 10; i++) {
			t2 -= input.nextInt() *20;
		}
		
		if(t1 <= t2){
			System.out.println("No chance.");
		}else{
			System.out.println("Dirty debug :(");
		}
	}
}

File: timus/python/p1636.py
# Problem 1636. Penalty Time
# http://acm.timus.ru/problem.aspx?space=1&num=1636

t1, t2 = map(int,raw_input().split())

rejected_times = map(int,raw_input().split())

for rejected in rejected_times:
	if rejected >0:
		t1 += 20*rejected

if t1 > t2:
	print "Dirty debug :("
else:
	print "No chance."
